lib/backend/syncersv1/updateprocessors/configurationprocessor.go

```diff
@@ -230,6 +231,13 @@ func (c *configUpdateProcessor) processAddOrModified(kvp *model.KVPair) ([]*mode
 				switch vt := value.(type) {
 				case string:
 					value = vt
+				case v1.Duration:
+					switch fieldInfo.Tag.Get("configv1timescale") {
+					case "seconds":
+						value = fmt.Sprintf("%.1f", vt.Seconds())
+					case "milliseconds":
+						value = fmt.Sprintf("%.1f", float64(vt.Duration/time.Millisecond))
+					}
```

Comment:
Bit worried that it'd be easy to leave this tag out when adding a new field.  Should we default to seconds?

Reply:
I think there is only one field that uses ms, so it would make some sense.

Reply:
Yea good catch.